.github/workflows/Tests.yaml

```diff
@@ -552,6 +556,23 @@ jobs:
           python3 -m pip install pybind11~=2.6.1
           python3 -m pip install -r support/Python/requirements.txt \
             -r support/Python/dev_requirements.txt
+      - name: Install ParaView
+        if: matrix.test_3d_rendering == 'ON'
+        working-directory: /work
+        # Can't just `apt-get install python3-paraview` because we need the
+        # headless build of ParaView. So we download a binary from paraview.org
+        # (which is built for a specific Python version unfortunately).
+        run: |
+          apt-get install -y libglvnd-dev  # Needed to find 'libglapi.so'
+          wget -O paraview.tar.gz --no-check-certificate "https://www.paraview.org/paraview-downloads/download.php?submit=Download&version=v5.10&type=binary&os=Linux&downloadFile=ParaView-5.10.1-osmesa-MPI-Linux-Python3.9-x86_64.tar.gz"
+          tar -xzf paraview.tar.gz
+          rm paraview.tar.gz
+          mv ParaView-* /opt/paraview
+          # Make 'paraview' Python package available
+          PYTHONPATH=/opt/paraview/lib/python3.9/site-packages:$PYTHONPATH
+          # Give system-installed Python packages priority over ParaView's
+          PYTHONPATH=/usr/local/lib/python3.9/dist-packages:$PYTHONPATH
```

Comment:
Just put the paraview python at the end of `$PYTHONPATH`?

Also, in the gcc-10 Debug build, looks like an H5 python test is using the `h5py` from paraview and not from the system. So I imagine that the paraview python is maybe used for other tests as well and I don't think we want that.

Reply:
> Just put the paraview python at the end of $PYTHONPATH?

Doesn't work because the `PYTHONPATH` is empty by default and system locations are searched automatically, so to give system packages priority you have to prepend them to the path.